pkgs/desktops/gnome/core/gnome-shell/default.nix

```diff
@@ -155,6 +156,7 @@ stdenv.mkDerivation rec {
     gnome-settings-daemon
     gobject-introspection
     mesa
+    libgda
```

Comment:
Rather than adding more dependencies to GNOME Shell, we patch the extensions: https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-common-issues-unwrappable-package-gnome-shell-ext

Reply:
Thanks for the pointers. Done.